schemas/glean/glean/glean-min.1.schema.json

```diff
@@ -0,0 +1,630 @@
+          "type": "string"
+        },
+        "architecture": {
+          "description": "The architecture of the device, (e.g. \"arm\", \"x86\").",
+          "type": "string"
+        },
+        "build_date": {
+          "description": "The date & time the application was built",
+          "format": "datetime",
+          "type": "string"
+        },
+        "client_id": {
+          "description": "A UUID uniquely identifying the client.",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "type": "string"
+        },
+        "device_manufacturer": {
+          "description": "The manufacturer of the device the application is running on. Not set if the device manufacturer can't be determined (e.g. on Desktop).",
+          "type": "string"
+        },
+        "device_model": {
+          "description": "The model of the device the application is running on. On Android, this is Build.MODEL, the user-visible marketing name, like \"Pixel 2 XL\". Not set if the device model can't be determined (e.g. on Desktop).",
+          "type": "string"
+        },
+        "first_run_date": {
+          "description": "The date of the first run of the application.",
+          "format": "datetime",
+          "type": "string"
+        },
+        "locale": {
+          "description": "The locale of the application during initialization (e.g. \"es-ES\"). If the locale can't be determined on the system, the value is [\"und\"](https://unicode.org/reports/tr35/#Unknown_or_Invalid_Identifiers), to indicate \"undetermined\".",
+          "type": "string"
+        },
+        "os": {
+          "description": "The name of the operating system. Possible values: Android, iOS, Linux, Darwin, Windows, FreeBSD, NetBSD, OpenBSD, Solaris, unknown",
+          "type": "string"
+        },
+        "os_version": {
+          "description": "The user-visible version of the operating system (e.g. \"1.2.3\"). If the version detection fails, this metric gets set to `Unknown`.",
+          "type": "string"
+        },
+        "session_count": {
+          "description": "An optional running counter of the number of sessions for a client.",
+          "type": "integer"
+        },
+        "session_id": {
+          "description": "An optional UUID uniquely identifying the client's current session.",
+          "type": "string"
+        },
+        "telemetry_sdk_build": {
+          "description": "The version of the Glean SDK",
+          "type": "string"
+        },
+        "windows_build_number": {
+          "description": "The optional Windows build number, reported by Windows (e.g. 22000) and not set for other platforms",
+          "type": "integer"
+        }
+      },
+      "required": [
+        "app_build",
+        "app_display_version",
+        "architecture",
+        "first_run_date",
+        "os",
+        "os_version",
+        "telemetry_sdk_build"
+      ],
```

Comment:
Should these all still be required in the minimal scheme?
I _think_ right now this schema would enforce that either it has no `client_info` or if it exists it has the required fields at least. Is that what we want? Or would we allow a smaller `client_info` too?

Reply:
Given the conversation above, I've removed these from the `glean-min` schema. If we want to progressively add info properties we will be able to do that in an adhoc manner directly to the minimum schema.